src/usr/local/www/system_camanager.php

```diff
@@ -107,6 +108,7 @@
 	if (!empty($a_ca[$id]['prv'])) {
 		$pconfig['key'] = base64_decode($a_ca[$id]['prv']);
 	}
+	$pconfig['trusted'] = $a_ca[$id]['trusted'];
```

Comment:
Think this is what you really want to do here: `$pconfig['trusted']  = isset($a_ca[$id]['trusted']);`